### PR TITLE
Add fingerprintjs2 library

### PR DESCRIPTION
**Pull request by Valve**

### CODE REVIEW

File: files/fingerprintjs2/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Valentin Vasilyev"
+github = "https://github.com/Valve/fingerprintjs2"
+homepage = "http://valve.github.io/fingerprintjs2/"
+description = "Modern & flexible browser fingerprinting library, a successor to the original fingerprintjs"
+mainfile = "dist/fingerprint2.min.js"
```
Comment (jimaek) [inline files/fingerprintjs2/info.ini]: Dont use dist/ here. Just `fingerprint2.min.js`

File: files/fingerprintjs2/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "fingerprintjs2",
+  "repo": "Valve/fingerprintjs2",
+  "files": {
+    "include": ["fingerprint2.js", "dist/fingerprint2.min.js"]
```
Comment (jimaek) [inline files/fingerprintjs2/update.json]: I dont think that `fingerprint2.js` is required in production. Can you remove it?

**Comment by Valve** [comment]
@jimaek done, please check out

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
Thank you

**Comment by Valve** [comment]
thanks!

**Comment by Valve** [comment]
@jimaek , seems like something got wrong, because I still don't see the lib on the CDN.

**Comment by jimaek** [comment]
It deploys instantly to the CDN but the website takes a while to update. It will be fixed in the new website.

here it is now http://www.jsdelivr.com/#!fingerprintjs2

**Comment by Valve** [comment]
Thanks!

**Pull request closed without merging**